airbyte-webapp/src/components/connection/CatalogTree/StreamHeader.module.scss

```diff
@@ -58,7 +58,8 @@
 }
 
 .purpleBackground {
-  background-color: colors.$blue-transparent;
+  background-color: colors.$blue;
+  opacity: 0.1;
```

Comment:
This won't have the same effect and could be risky in other areas where the colors are used. There are other uses of opacity in the rest of the app. I wonder if this would work... maybe using the actual CSS var or situations like these?

https://www.bmorecreativeinc.com/rgb-values-as-css-variables/
```suggestion
  background-color: rgba(var(--color-blue), 0.1); ?
```

Reply:
Oops, good catch! Yes we can actually even use `background-color: rgba(colors.$blue, .1)` - that should work fine.

Reply:
Actually on second thought, I've changed this to `background-color: colors.$blue-40`. I think this should be basically imperceptible, and it means we don't need this opaque version of `blue-400` anymore. WDYT?

![image](https://user-images.githubusercontent.com/7550957/201698229-a4738a9a-e844-4cb1-b588-1448d4a0b1ea.png)